### ISSUE TITLE
Synced appointment gets visible in calendar view only after clicking hide followed by unhide in the calendar pane

### ISSUE THREAD

**Issue by iusgit** [jobisoft/EAS-4-TbSync#153]
## Your environment
MacOSX 11.6.1
TbSync version: 3.0.1
EAS-4-TbSync version: 2.0.1
Thunderbird version: 91.3.1

## Expected behavior
After sync'ing the Office365 Teams calendar all appoinments should show up in the calendar view, date shifts etc. included.

## Actual behavior
Synced appointments gets visible in calendar view only after clicking hide followed by unhide (eye symbol) in the calendar pane. 

## Steps to reproduce
1. Create appointment in Teams / Office 365.
2. Synchronize in TB
3. Modify date of appoinment in Teams / Office 365.
4. Synchronize in TB (no change visible)
5. Click Hide/Unhide (changes visible

To help resolving your issue, enable debug logging (TbSync Account Manager -> Help) and send me the debug.log via e-mail (use the title of your issue as subject of the email).

**Comment by StianOby** [comment]
I'm seeing the same behavior. Thanks for identifying a "work-around" in the form of hide/unhide. Perhaps #154 is the same issue?

**Comment by ghost-from-the-past** [comment]
any action that refresh the calendar tab do the job
seems like a canvas refresh is missing somewhere

**Comment by jobisoft** [comment]
The project has moved on to a total rewrite, the new official and only supported version on GitHub is v5:

https://github.com/jobisoft/TbSync/releases
https://github.com/jobisoft/EAS-4-TbSync/releases

If you can reproduce the issue on v5, please create a new issue with some debug info. Thank you for your support.
